platform/viewer/public/config/local_static.js

```diff
@@ -1,8 +1,14 @@
 window.config = {
   routerBasename: '/',
-  // whiteLabelling: {},
-  extensions: [],
-  modes: [],
+  // Don't list the extensions specifically - let the config mode define them
+  extensions: ['@ohif/extension-tmtv', '@radicalimaging/ecg-dicom', 'unknown'],
```

Comment:
@swederik and @sedghi  - here is an example of loading the tmtv extension and the ecg dicom extensions.
The tmtv extension comes from the package.json linkage, while the ecg-dicom one comes from the Radical Imaging provided extension.  The viewer will load fine without either of them.